src/coreComponents/common/logger/Logger.hpp

```diff
@@ -456,20 +456,23 @@
  * @brief Output messages based on current Group's log level.
  * @param[in] minLevel minimum log level
  * @param[in] msg a message to log (any expression that can be stream inserted)
+ * @deprecated Will be replaced by GEOS_LOG_LEVEL_INFO
```

Comment:
shall we create an issue to remind ourselves to remove these at some point?

Reply:
[Done !](https://github.com/GEOS-DEV/GEOS/issues/3385) :)